base-helm-configs/memcached/memcached-helm-overrides.yaml

```diff
@@ -573,7 +566,7 @@ serviceAccount:
 persistence:
   ## @param persistence.enabled Enable Memcached data persistence using PVC. If false, use emptyDir
   ##
-  enabled: true
+  enabled: false
```

Comment:
Why change this? Memcached operates from memory no matter what, the persistence option here makes it possible to automatically recover the cache in the case of a pod failover; which has proven operational benefits.

Reply:
I don't see any benefit in that as I don't allow memcache to roam in my deployment so I guess that becomes debatable what the default should be

Reply:
I think the default should be whatever provides the most resiliance; especially as the cloud scales.